non-protocol-specs/0011-NP-CLIE-client-interaction-tests.md

```diff
@@ -18,3 +18,14 @@ This specification contains a set of tests/acceptance criteria that clients (wal
     - This is communicated to the wallet, and wallet uses the new parameter for each transaction tied to a block with a height higher than the one in which the change happened. This requires the wallet to be subjected to a difficulty increase due to too many messages (<a name="0011-NP-CLIE-005" href="#0011-NP-CLIE-005">0011-NP-CLIE-005</a>)
 6. The parameter `spam.pow.increaseDifficulty` is changed from `1` to `0`.  Verify that
     - This is communicated to the wallet, and wallet uses the new parameter for each transaction tied to a block with a height higher than the one in which the change happened. The wallet should submit no more than `spam.pow.numberofTxPerBlock` PoWs tied to a given block before using another. If this results in none being available the transaction should not be sent. (<a name="0011-NP-CLIE-006" href="#0011-NP-CLIE-006">0011-NP-CLIE-006</a>)
+7. Link more than <SpamPoWNumberOfTxPerBlock>. (currently, 2) transactions to a single block for the PoW; assure that the client does not allow that and the account does not get blocked. (<a name="0011-NP-CLIE-007" href="#0011-NP-CLIE-007">0011-NP-CLIE-007</a>)
```

Comment:
"Link more than <SpamPoWNumberOfTxPerBlock>" doesn't work from the client point of view. 

The whole PoW should be largely transparent to a client. 

A test can be more along the lines: set `spam.pow.numberOfPastBlocks = 100`. Take a client who sends 400 transactions as quickly as possible to the wallet. The wallet will correctly compute what PoW difficulty must be attached to which block historical block and submits these as quickly as possible without getting the party banned.

Also, < xxx > doesn't do any markdown formatting (since you wanted a brutal review).